src/components/AppBar/index.scss

```
@@ -0,0 +1,63 @@
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@import '@material/tab/mixins';
+@import '@material/tab-indicator/mixins';
+
+.AppBar {
+  padding: 0 24px;
```

```suggestion
  padding: 0 1.5rem;
```

To align with `rem`s used in the rmwc components.

---

My thought is that we actually should not use rem's in the core, we use px across the Console and we should stay consistent with that spec. _if_ we want to revisit this, someone could easily come back and write a script to update these all...